forte_wrapper/allennlp/allennlp_processors.py

```diff
@@ -210,3 +214,30 @@ def _create_srl(input_pack: DataPack, tokens: List[Token],
                                         tokens[arg_span.end].end)
                 link = PredicateLink(input_pack, pred, arg)
                 link.arg_type = label
+
+    @classmethod
+    def expected_types_and_attributes(cls) -> Dict[str, Set[str]]:
+        r"""Method to add expected type for current processor input which
+        would be checked before running the processor if
+        :meth:`~forte.pipeline.Pipeline.enforce_consistency` was enabled for
+        the pipeline.
+        """
+        expectation_dict: Dict[str, Set[str]] = {
+             "ft.onto.base_ontology.Sentence": set()
+        }
+        return expectation_dict
+
+    def record(self, record_meta: Dict[str, Set[str]]):
+        r"""Method to add output type record of current processor
+        to :attr:`forte.data.data_pack.Meta.record`.
+
+        Args:
+            record_meta: the field in the datapack for type record that need to
+                fill in for consistency checking.
+        """
+        if "tokenize" in self.configs.processors:
+            record_meta["ft.onto.base_ontology.Token"] = set()
+            if "pos" in self.configs.processors:
+                record_meta["ft.onto.base_ontology.Token"].add("pos")
+            if "depparse" in self.configs.processors:
+                record_meta["ft.onto.base_ontology.Dependency"] = set()
```

Comment:
depparse also contains `rel_type` as attribute right, so
```suggestion
                record_meta["ft.onto.base_ontology.Dependency"] = {"rel_type"}
```